include/boost/math/tools/mp.hpp

```diff
@@ -335,6 +335,108 @@ using mp_remove_if = typename detail::mp_remove_if_impl<L, P>::type;
 template<typename L, typename Q> 
 using mp_remove_if_q = mp_remove_if<L, Q::template fn>;
 
+// Index sequence
+// Use C++14 index sequence if available
+#ifndef BOOST_NO_CXX14_VARIABLE_TEMPLATES
+#include <utility>
+template<std::size_t... I>
+using index_sequence = std::index_sequence<I...>;
```

Comment:
@jzmaddock `std::index_sequence` is a C++14 feature, but I didn't see anything in config that specifically called it out. Is there something better to put here?

Reply:
Try `defined(__cpp_lib_integer_sequence) && (__cpp_lib_integer_sequence >= 201304)`

Reply:
That does the trick locally, hopefully CI agrees. Thanks.